src/makeCalculator.js

```diff
@@ -4,7 +4,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(value) {
+      this.result += value;
+
+      return this;
+    },
+
+    subtract(value) {
+      this.result -= value;
+
+      return this;
+    },
+
+    multiply(value) {
+      this.result *= value;
+
+      return this;
+    },
+
+    divide(value) {
+      this.result /= value;
+
+      return this;
+    },
+
+    reset(value) {
+      this.result = 0;
+
+      return this;
```

Comment:
The `reset` method takes a `value` parameter, but it is not used in the method. If the intention is to reset the result to a specific value, you should assign `this.result = value;`. Otherwise, you can remove the `value` parameter if it's not needed.